app/test_main.py

```diff
@@ -1,3 +1,25 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,result",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (28, 28, [3, 2]),
+        (-1, -11, [0, 0])
```

Comment:
Add test cases with large numbers, numbers different for `cat_age` and `dog_age`, and test for exceptions when invalid arguments are passed, such as tuples, strings, dicts, e.g. as mentioned in the checklist. You could create a separate function to test against raised exceptions.